server/data.js

```diff
@@ -0,0 +1,33 @@
+    {id:23, category :'social', title:'Social Enterprise UK | Home', url:'https://www.socialenterprise.org.uk/', description:'The UK body for social enterprise - business with a social or environmental mission. We support business where society profits.'},
+    {id:24, category :'social', title:'Social | Definition of Social by Merriam-Webster', url:'https://www.merriam-webster.com/dictionary/social', description:'social · 1 : relating to or involving activities in which people spend time talking to each other or doing enjoyable things with each other She has poor/good ...'},
+    {id:25, category :'social', title:'Why we may have to re-learn to socialise - BBC Worklife', url:'https://www.bbc.com/worklife/article/20210218-why-we-may-have-to-re-learn-to-socialise', description:'18 Feb 2021 — With socialising at a near halt during the pandemic, our social muscles have atrophied. We may have to re-train ourselves to be with other'},    {id:26, category :'social', title:'Impact of the COVID-19 pandemic on social media - Wikipedia', url:'https://en.wikipedia.org/wiki/Impact_of_the_COVID-19_pandemic_on_social_media', description:"During a time of social distance and limited contact with others, social media became an important place to interact. Social media platforms are meant to ..."},
+    {id:27, category :'social', title:'Social media - Wikipedia', url:'https://en.wikipedia.org/wiki/Social_media', description:'Some of the most popular social media websites, with over 100 million registered users, include Facebook (and its associated Facebook Messenger), TikTok, WeChat ...'},
+    {id:28, category :'social', title:'The Top 10 Social Media Sites & Platforms 2021', url:'https://www.searchenginejournal.com/social-media/biggest-social-media-sites/', description:'22 Jun 2021 — The Top 10 Social Media Sites and Platforms · 1. Facebook · 2. YouTube · 3. WhatsApp · 4. Instagram · 5. TikTok · 6. Snapchat · 7. Reddit · 8. Pinterest.'},
+    {id:29, category :'social', title:'9 Types of Social Media and How Each Can Benefit Your Business', url:'https://blog.hootsuite.com/types-of-social-media/', description:'2 Sept 2021 — One thing that\'s changed since the early days of social media is that many platforms used to focus on one function, such as social networking or ...'},
+    {id:30, category :'social', title:'What is Social Media? - WhatIs.com', url:'https://whatis.techtarget.com/definition/social-media', description:'Social media is a collective term for websites and applications that focus on communication, community-based input, interaction, content-sharing and collaboration. ... Some popular examples of general social media platforms include Twitter, Facebook and LinkedIn.'}
+]
+
+module.exports = queries
```

Comment:
Maybe you could have nested the objects inside of a root object rather than storing them in an array